key_action_map.cpp

```diff
@@ -120,6 +120,43 @@ key_action_map get_random_kam()
       );
 }
 
+std::vector<key_action_map> get_n_random_kams(int n)
+{
+  std::vector<key_action_map> vector_kam;
```

Comment:
To be examplary, add `assert(n >= 0);`, else the for loop will do approx 2^64 too much iterations :-)

Reply:
```suggestion
  assert(n >= 0);
  std::vector<key_action_map> vector_kam;
```